tox.ini

```diff
@@ -151,6 +148,25 @@ deps=https://github.com/twisted/pydoctor/archive/3f9c64829dfa040b334c9ae27c332c7
 [testenv:manifest-checker]
 skip_install = true
 
+[testenv:lint]
+skip_install=True
+basepython=python3.6
```

Comment:
I'm not in favor of pinning the lint target to Python 3.6,
because `tox -e lint` will not work for contributors who do not have Python 3.6 installed.

Reply:
I don't really understand the motivation for this change myself. Perhaps the goal was to prevent linting tools from running under Python 2.7 if Tox was installed with that Python version? I'll remove it.